docs/performance_data.md

```diff
@@ -0,0 +1,21 @@
+Performance Data
+================
+
+AWX has the ability to collect performance data on job runs.
+
+The following data is collected periodically (by default, every 0.25 seconds):
```

Comment:
> (by default, every 0.25 seconds)

might be more clear to say "with a default interval of 0.25 seconds).

The data is not collected by default (given default verbosity is not 5).

Reply:
good catch, updated